### PR TITLE
Fixes for issues found by coverity scan

### PR DESCRIPTION
**Pull request by vojtechtrefny**

### CODE REVIEW

**Comment by vpodzime** [comment]
Jenkins, test this, please.

File: tools/udisksctl.c [hunk 1]
```diff
@@ -943,8 +943,6 @@ read_passphrase (void)
   str = NULL;
 
  out:
-  if (str != NULL)
-    g_string_free (str, TRUE);
   return ret;
```
Comment (vpodzime) [inline tools/udisksctl.c]: I'd prefer the ``goto``s above to be replace by ``return``s with the ``out`` label removed instead. But YMMV.

Comment (vojtechtrefny) [reply tools/udisksctl.c]: Fixed.

**Comment by vpodzime** [comment]
Jenkins, test this, please.

**Pull request merged**